seismic/receiver_fn/c_sippl_reference/HK/LN.SNY.rrf_85_85.list

```diff
@@ -1,559 +0,0 @@
-2001.001_06.57.LN.SNY.BHRRF.SAC   8.63000   0.10  174.718   34.817
```

Comment:
Where this external package will be stored then?

Reply:
Alexei and I both have a copy of this on NCI. I we want to keep a copy of it checked in somewhere, then it should be done as a separate github repository.